pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/PostQueryCommand.java

```diff
@@ -82,6 +86,11 @@ public PostQueryCommand setBrokerPort(String port) {
     return this;
   }
 
+  public PostQueryCommand setBrokerProtocol(String protocol) {
+    _brokerProtocol = protocol;
```

Comment:
Validate the protocol

Reply:
this directly opens a HttpURLConnection. If the protocol/schema is invalid, it immediately calls it out.